src/common.js

```diff
@@ -111,6 +115,7 @@ exports.ShellString = ShellString;
 
 // Return the home directory in a platform-agnostic way, with consideration for
 // older versions of node
+// XXX: I don't think this should be a part of plugin.utils. Not sure what should happen to it. Maybe just inline it.
```

Comment:
Isn't that what `common.js` is for--common utilities that most files depend on in some way? Since wrap uses this, why bother moving it out? Let me know what your thoughts are.